src/layered_config_tree/main.py

```diff
@@ -358,20 +358,20 @@ def get(self, key: str, default_value: NodeValue = None) -> LayeredConfigTree |
         """
         return self[key] if key in self._children else default_value
 
-    def get_tree(self, key: str, default_value: NodeValue = None) -> LayeredConfigTree:
+    def get_tree(self, key: str) -> LayeredConfigTree:
         """Return the LayeredConfigTree at the key in the outermost layer of the config tree.
 
         Parameters
         ----------
         key
             The str we look up in the outermost layer of the config tree.
         """
-        data = self.get(key, default_value)
+        data = self.get(key)
```

Comment:
We probably want this to be 
```python
data = self[key]
```
since we want to throw an error if the key is missing anyway